frontend/pages/hosts/details/cards/HostSummary/MacSettingsIndicator/MacSettingsIndicator.tsx

```diff
@@ -29,24 +34,34 @@ const STATUS_DISPLAY_OPTIONS: StatusDisplayOptions = {
     iconName: "pending-partial",
     tooltipText: "Host will apply the latest settings when it comes online",
   },
-  Failing: {
+  Failed: {
     iconName: "error",
     tooltipText:
       "Host failed to apply the latest settings. Click to view error(s).",
   },
 };
 
+/**
+ * Returns the displayed status of the macOS settings field based on the
+ * profile statuses.
+ * If any profile has a status of "failed", the status will be displayed as "Failed" and
+ * continues to fall through to "Pending" and "Verifying" if any profiles have those statuses.
+ * Finally if all profiles have a status of "verified", the status will be displayed as "Verified".
+ */
 const getMacSettingsStatus = (
   hostMacSettings?: IHostMacMdmProfile[]
 ): MacSettingsStatus => {
   const statuses = hostMacSettings?.map((setting) => setting.status);
   if (statuses?.includes("failed")) {
-    return "Failing";
+    return "Failed";
   }
   if (statuses?.includes("pending")) {
     return "Pending";
   }
-  return "Verifying";
+  if (statuses?.includes("verifying")) {
+    return "Verifying";
+  }
+  return "Verified";
```

Comment:
To avoid false positives and surface potential issues, I wonder if it would be good to build in a default case? For example, this would return "Verified" if statuses is `undefined`, which may not be the expected behavior.

```ts
if (statuses?.every((s) => s === "verified") {
    return "Verified";
}
return "Unknown";
```

Reply:
I've changed the function signature so that we avoid false positives. Now at this point in the code
we should always have profiles with statuses.